common/djangoapps/student/models/user.py

```diff
@@ -1398,11 +1398,21 @@ def add_to_profile_url(self, course_name, cert_mode, cert_url, certificate=None)
                 If provided, this function will also autofill the certId and issue date for the cert.
         """
         params = {
-            'name': self._cert_name(course_name, cert_mode),
+            'name': self._cert_name(course.display_name, cert_mode),
             'certUrl': cert_url,
         }
 
-        params.update(self._organization_information())
+        # By default when sharing to LinkedIn, Platform Name and/or Platform LINKEDIN_COMPANY_ID will be used.
+        # If Course specific Organization Name is prefered when sharing Certificate to linkedIn the flag for that
+        # CERTIFICATE_LINKEDIN_DEFAULTS_TO_COURSE_ORGANIZATION_NAME should be set to True alongside other LinkedIn settings
+        share_settings = configuration_helpers.get_value('SOCIAL_SHARING_SETTINGS', settings.SOCIAL_SHARING_SETTINGS)
```

Comment:
we shouldn't make the same configuration getter call  multiple times in the same process, and the line 

```
share_settings = configuration_helpers.get_value('SOCIAL_SHARING_SETTINGS', settings.SOCIAL_SHARING_SETTINGS)
```

is already called 3 times in this class. Could you make a class init that sets this on `self` so we only have to call it once?